content/pages/docs-as-code.html

```diff
@@ -8,7 +8,7 @@
     <meta name="template" content="page" />
     <meta name="slug" content="docs-as-code" />
     <meta name="status" content="hidden" />
-    <meta name="description" content="Use docs as code pattern with Read the Docs and supercharge your docs" />
+    <meta name="description" content="Our docs as code hosting is used by thousands of projects successfully. Don't reinvent the wheel." />
```

Comment:
I'd be more descriptive than "successfully" here, it sounds a bit like a low bar to clear.